resources/web/template.html

```diff
@@ -164,72 +162,6 @@
         background-size: contain;}
     </style>
 
-    <script type="text/javascript" src="/static/js/horizon-swiper.min.js"></script>
-
-    <script type="text/javascript">
-      $(document).ready( function(){
-        // if($('#scrollable-pane li').length > 3){
-          $('.horizon-swiper').horizonSwiper({
-            showItems: 3,
-            arrows: true
-          }); 
-        // }
-        
-        $('.navbar-toggle').on('click', function(){
-          $('.mobile-navigation, .menu-open').hide();
```

Comment:
I searched for all of the classes and id mentioned in this section and I don't see any of them.